/home/exouser/YuZheng/experiments/autoencoder_kl_gan/world_size1-hmdb_autoencoder
load yaml from config
Launching processes...
2025-02-06 03:21:54,471 train INFO: Building config ...
2025-02-06 03:21:54,471 train INFO: Building dataloaders ...
Total frames in dataset: 186436
2025-02-06 03:23:05,346 train INFO: Train dataloaders build complete
Total frames in dataset: 186436
2025-02-06 03:24:21,523 train INFO: Valid dataloaders build complete
2025-02-06 03:24:21,523 train INFO: Building models ...
Traceback (most recent call last):
  File "/home/exouser/YuZheng/train.py", line 179, in <module>
    main(args)
  File "/home/exouser/YuZheng/train.py", line 153, in main
    subprocess_fn(args)
  File "/home/exouser/YuZheng/train.py", line 57, in subprocess_fn
    model = builder.get_model()
  File "/home/exouser/YuZheng/utils/builder.py", line 86, in get_model
    model = autoencoder_kl_gan_model(self.logger, **params)
  File "/home/exouser/YuZheng/models/autoencoder_kl_gan_model.py", line 20, in __init__
    super().__init__(logger, **params)
  File "/home/exouser/YuZheng/models/model.py", line 71, in __init__
    self.checkpoint_ceph = checkpoint_ceph(checkpoint_dir=self.ceph_checkpoint_path)
  File "/home/exouser/YuZheng/utils/checkpoint_ceph.py", line 16, in __init__
    self.client = Client(conf_path=conf_path)
NameError: name 'Client' is not defined
